src/components/shared/footer.tsx

```diff
@@ -1,16 +1,17 @@
 import { css } from '@emotion/core';
-import { Footer } from '@guardian/src-footer';
-import { brandBorder, brandText, space } from '@guardian/src-foundations';
+import { brandBorder, neutral, space } from '@guardian/src-foundations';
 import { textSans } from '@guardian/src-foundations/typography';
 import { bool } from 'aws-sdk/clients/signer';
 import React, { FC } from 'react';
 
 
 // Footer content styles
 const container = css`
-    border-style: solid;
-    border-color: ${brandBorder.primary};
     border-width: 0 1px;
+    ${textSans.small({ lineHeight: "regular" })};
+    margin-left: 10px;
+    margin-right: 10px;
+    margin-bottom: 10px;
```

Comment:
The left/right margin for the main article body is `.5rem` which is remSpace[2] in the design system I think

Reply:
do we use `remspace` for the top and bottom margins also?

Reply:
Ideally we always use `rem`s unless there's a specific reason to use pixels, otherwise the UI will break when scaled.

In this case I think we should also try to match this to the closest spacing value available in the design system, which is `remSpace[2]` (equivalent to `8px`) or `remSpace[3]` (equivalent to `12px`).